.circleci/config.yml

```diff
@@ -1,23 +1,27 @@
-# Python CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-python/ for more details
-#
 version: 2
 jobs:
   test:
     docker:
-      - image: cimg/go:1.14.1
+      - image: cimg/go:1.19
+        environment:
+          GOFLAGS: -p=8
```

Comment:
```	-p n
		the number of programs, such as build commands or
		test binaries, that can be run in parallel.
		The default is GOMAXPROCS, normally the number of CPUs available.
```

Reply:
Do you think we should change it? I think the circleci will be fine.